themes/westerndevs/scripts/author-list-json.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+hexo.extend.generator.register('author-list-json', function(locals) {
+
+  var authorList = [];
+  var authorKeys = Object.keys(locals.data.authors);
+  for (var i = 0; i < authorKeys.length; i++) {
```

Comment:
I tried `for( var authorKey in authorKeys )` but they just gives me a list of indices rather than actual values.